<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SharedFanControlDialog</class>
 <widget class="QDialog" name="SharedFanControlDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="fanFaultStateLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>80</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>风机控制选择</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>25</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>自动</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fanOpenPushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>风机运行</string>
   </property>
  </widget>
  <widget class="QComboBox" name="fanIndexComboBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>430</y>
     <width>110</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>点击切换手自动模式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fanStopPushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>风机停止</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>风机状态</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>运行状态</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>25</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>手动</string>
   </property>
  </widget>
  <widget class="QPushButton" name="switchFanControlModePushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>350</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="fanRunStateLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="fanIndexLabel">
   <property name="geometry">
    <rect>
     <x>189</x>
     <y>80</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>故障状态</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string># 风机控制</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>备用风机状态</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>运行状态</string>
   </property>
  </widget>
  <widget class="QLabel" name="backupFanRunStateLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="backupFanOpenPushButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>240</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>启用备用风机</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backupFanStopPushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>240</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>停用备用风机</string>
   </property>
  </widget>
  <widget class="QLabel" name="outputFreqLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>300</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>300</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>当前频率</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>296</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>输出频率选择</string>
   </property>
  </widget>
  <widget class="MyCustomSlider" name="outputFreqSlider">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>340</y>
     <width>251</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>300</y>
     <width>20</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Hz</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyCustomSlider</class>
   <extends>QSlider</extends>
   <header location="global">mycustomslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
